src/components/home/Banner.jsx

```diff
@@ -12,49 +12,56 @@ function Banner(props) {
   const { isMobile, location } = props;
   return (
     <div className="banner-wrapper">
-      {isMobile && (
-        <TweenOne animation={{ opacity: 1 }} className="banner-image-wrapper">
-          <div className="home-banner-image">
-            <img
-              alt="banner"
-              src="https://gw.alipayobjects.com/zos/rmsportal/rqKQOpnMxeJKngVvulsF.svg"
-              width="100%"
+      <div className='banner-top-image'>
+        {isMobile && (
```

Comment:
I think it has to do either with:

1. The race condition if `isMobile` here and `!isMobile` at the bottom 
2. Something with `TweenOne` or `QueueAnim` where they are overwriting each others properties in some way when it's ambiguous.